magma/home/__init__.py

```diff
@@ -9,7 +9,7 @@
 except ImportError:
     from io import StringIO
 
-from flask import Blueprint, request, render_template, make_response
+from flask import Blueprint, flash, request, render_template, make_response, session
```

Comment:
Usually, if I think about it I'll leave a trailing comma. It's pretty common practice in python to do that for any list, especially those you might end up adding to later. Not required though in any sense.